### PR TITLE
WIP: inkscape: update to 1.1

### PR DESCRIPTION
**Pull request by atk**
#### General
- [ ] This is a new package and it conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements)

#### Have the results of the proposed changes been tested?
- [x] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me
- [ ] I generally don't use the affected packages but briefly tested this PR

### CODE REVIEW

**Comment by Chocimier** [comment]
Cross has to work, try rust build helper.

**Comment by atk** [comment]
The glyph tests are failing due to a font config misconfiguration that stop the arabic/chinese characters to be rendered correctly.

**Comment by atk** [comment]
There seemed to be another issue. An included font was referenced by the wrong name. I'm just trying to run the tests locally again.

**Comment by atk** [comment]
I believe the remaining issues are down to UTF or font misconfiguration. Not sure what it is, but investigation continues.

File: srcpkgs/inkscape/template [hunk 1]
```diff
@@ -35,4 +34,8 @@ post_patch() {
 		# since inkscape uses NULL instead of nullptr
 		vsed -e "/-Werror=format/d" -i CMakeScripts/DefineDependsandFlags.cmake
 	fi
+	# make test font available under the name it is used with
+	pushd testfiles/rendering_tests/fonts/
+	ln -s GeomTest-Regular.otf GeomTest.otf
+	popd
```
Comment (ericonr) [inline srcpkgs/inkscape/template:40]: Just use `cd`, it should only happen inside the function...

**Review by ericonr — COMMENTED** [review]
The aarch64 build failure shows:

```
2021-06-04T19:54:19.3688833Z /usr/lib/gcc/aarch64-linux-gnu/10.2.1/../../../../aarch64-linux-gnu/bin/ld: cannot find -latomic_ops
```

**Comment by atk** [comment]
Tried to fix the template and aarch64 issues, having another look at the fonts issue later.

**Comment by atk** [comment]
I have to admit I'm a bit stumped. I added the dependency of libatomic_ops-devel, which should have resolved the issue, but it did not. Any idea how that could happen?

**Comment by atk** [comment]
Now on to the last font issues. Installing the missing fonts as test dependencies should in theory fix this.

**Comment by atk** [comment]
That's strange: if I install all the missing fonts and run the tests, they fail locally, yet if I compare the expected output with the actual rendering, it's identical pixel by pixel. Should I simply disable the four failing tests for now?

File: srcpkgs/inkscape/template [hunk 2]
```diff
@@ -1,27 +1,27 @@
...
- glib-devel perl-XML-Parser tar which xz"
-makedepends="harfbuzz-devel libsoup-devel gsl-devel pango-devel
+ glib-devel perl-XML-Parser which"
+makedepends="harfbuzz-devel libsoup-devel gsl-devel pango-devel libatomic_ops-devel
  double-conversion-devel gc-devel libwpd-devel libcdr-devel libvisio-devel
  poppler-devel libwpd-devel potrace-devel gtkmm-devel gdl-devel gtkspell3-devel
  aspell-devel libxslt-devel libgomp-devel libwpg-devel poppler-glib-devel"
 depends="desktop-file-utils hicolor-icon-theme python3-lxml python3-numpy python3-scour"
-checkdepends="$depends gtest-devel ImageMagick cantarell-fonts"
+checkdepends="$depends gtest-devel ImageMagick cantarell-fonts noto-fonts-cjk-20190416_2
```
Comment (ericonr) [inline srcpkgs/inkscape/template:17]: Why are you specifying the version for `noto-fonts-cjk`? That will break whenever it's updated.

**Comment by ericonr** [comment]
Please pull in the depends change in #31336

File: srcpkgs/inkscape/template (hunk 2 above)
Comment (atk) [reply srcpkgs/inkscape/template:17]: I only used the package that I found for the font. Can I use `noto-fonts-cjk` instead?

Comment (ericonr) [reply srcpkgs/inkscape/template:18]: Yes.

**Comment by ericonr** [comment]
Please expand your commit message to explain the pillow change and that the package is being adopted.

File: srcpkgs/inkscape/template (hunk 2 above)
Comment (atk) [reply srcpkgs/inkscape/template]: ah, that works. Thanks!

**Comment by Chocimier** [comment]
There _are_ differences in kerning in test 299, maybe due to dependencies version. Disabling those test with comment seems fine.

**Comment by atk** [comment]
Now all that is left are some segfaults on x86_64-musl that I cannot explain. Maybe someone is able to help?

File: srcpkgs/inkscape/template [hunk 3]
```diff
@@ -35,4 +35,6 @@ post_patch() {
 		# since inkscape uses NULL instead of nullptr
 		vsed -e "/-Werror=format/d" -i CMakeScripts/DefineDependsandFlags.cmake
 	fi
+	# disable glyph tests that fail due to different hinting
+	vsed -e "s/\(test-glyph-y-pos\|test-rtl-vertical\|text-glyphs-combining\|text-glyphs-vertical\)/# \1 - skipped due to hinting issues/" -i testfiles/rendering_tests/CMakeLists.txt
```
Comment (ericonr) [inline srcpkgs/inkscape/template:39]: Can you split these into two or three lines? `vsed -e \` to make a multiline command.

File: srcpkgs/inkscape/template [hunk 4]
```diff
@@ -1,27 +1,27 @@
 # Template file for 'inkscape'
 pkgname=inkscape
-version=1.0.2
-revision=3
-wrksrc="inkscape-${version}_2021-01-15_e86c870879"
+version=1.1
+revision=1
+wrksrc="${pkgname}-${version}_2021-05-24_c4e8f9ed74"
```
Comment (ericonr) [inline srcpkgs/inkscape/template:5]: I don't think removing `inkscape` from here is good, I'd rather leave as is...

**Review by ericonr — COMMENTED** [review]
Segfaults on musl are known: https://gitlab.com/inkscape/inkscape/-/issues/2241

**Comment by hervyqa** [comment]
thank you for your hard work 👏👏👏 @atk @ericonr

**Pull request closed without merging**